frontend/src/components/common/page-container/styles.module.scss

```diff
@@ -0,0 +1,5 @@
+.container {
+  &.containerLarge {
+    max-width: 1670px;
```

Comment:
If I remember the daily discussion we decided to use the default lib container size